dataprofiler/profilers/profiler_options.py

```diff
@@ -367,6 +404,7 @@ def __init__(self) -> None:
         self.median_abs_deviation = BooleanOption(is_enabled=True)
         self.num_zeros = BooleanOption(is_enabled=True)
         self.num_negatives = BooleanOption(is_enabled=True)
+        self.num_quantiles = NumQuantilesOption(is_enabled=True)
```

Comment:
I'm wondering if this should be in `NumericalOptions`? We already have
`self.histogram_and_quantiles = HistogramOption()`.

Should we rename `HistogramOption` and insert it in that class?

Reply:
@clee1152 @taylorfturner , thoughts?

Reply:
I think this makes sense. Does this mean we will not go with your previous suggestion of renaming `NumQuantilesOption` to `QuantileOptions` and instead just insert it into `HistogramOption`?

Reply:
Fair Q. @taylorfturner may have more thoughts, but if we are keeping
`histogram_and_quantiles`
Then we should at least rename `HistogramOption` -> `HistogramAndQuantilesOption`
We could just move over the `num_quantiles` setting

Reply:
That sounds good to me! Will see what @taylorfturner thinks, then implement.